frontend/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -49,9 +49,12 @@ import compactSaveQueue from "oxalis/model/helpers/compaction/compact_save_queue
 import compactUpdateActions from "oxalis/model/helpers/compaction/compact_update_actions";
 import messages from "messages";
 import window, { alert, document, location } from "libs/window";
+import ErrorHandling from "libs/error_handling";
 
 import { enforceSkeletonTracing } from "../accessors/skeletontracing_accessor";
 
+const SLOW_SAVE_REQUEST_THRESHOLD = 5 * 1000; // 60 * 1000;
```

Comment:
The save interval is 30s, right? Shouldn't we log already if a single request takes longer than 30s, then? Otherwise the save "debt" will continue to build up which could lead to problems.

Reply:
Good point! I'll change the code to use the existing `PUSH_THROTTLE_TIME` (which is 30s).